manifests/init.pp

```diff
@@ -4,15 +4,15 @@
 #
 # === Parameters:
 #
-# $enable_yum::                                Enable rpm content plugin, including syncing of yum content
+# $enable_yum::                                Enable feature to sync, manage, and serve RPM content to be consumed by package managers like yum and dnf.
```

Comment:
What I liked about `Enable RPM content support` was that the relevant part was near the front. Now as a user you need to "skip" over half a sentence for the relevant bit. If you want to avoid support, you can try this:
```suggestion
# $enable_yum::                                Enable the RPM content feature. This allows syncing, managing and serving RPM content to be consumed by package managers like yum and dnf.
```

Reply:
Great points. This looks good to me and I'll make the change.